src/app/zap-templates/zcl/data-model/chip/service-area-cluster.xml

```diff
@@ -88,41 +88,41 @@ limitations under the License.
     </features>
 
     <!-- Attributes -->
-    <attribute side="server" code="0x0000" define="SupportedLocations"  type="array"     entryType="LocationStruct"  writable="false" isNullable="false" optional="false">SupportedLocations</attribute>
+    <attribute side="server" code="0x0000" define="SupportedAreas"      type="array"     entryType="AreaStruct"      writable="false" isNullable="false" optional="false">SupportedAreas</attribute>
     <attribute side="server" code="0x0001" define="SupportedMaps"       type="array"     entryType="MapStruct"       writable="false" isNullable="true"  optional="false">SupportedMaps</attribute>
```

Comment:
Pre-existing, but this is optional in the spec.  Separate PR?

Reply:
Yes. It will be fixed when addressing [this issue](https://github.com/project-chip/connectedhomeip/issues/34670).